test/functional/rpc_getblock_fee.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+'''Test getblock rpc fee calculation.
+'''
+
+import os
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import get_datadir_path
+
+
+class GetblockFeeTest(BitcoinTestFramework):
```

Comment:
I think `rpc_blockchain.py` is the natural place for these tests. A new file is not necessary for such a specific test IMO.